File: 6KYU/Multiples of 3 or 5/solution.rs
fn solution(num: i32) -> i32 {
    let mut total: i32 = 0;
    
    for i in 1..num {
        if i % 3 == 0 || i % 5 == 0 {
            total = total + i;
        }
    }
    return total;
}

File: 6KYU/Multiples of 3 or 5/solution.php
function solution($number){
  $total = 0;
  
  for ($x = 0; $x <= $number-1; $x++) {
    if($x % 3 == 0 || $x % 5 == 0) {
      $total = $total + $x;
    }
  }
    
  return $total;
}